public/app/plugins/panel/stat/types.ts

```diff
@@ -108,4 +109,32 @@ export function addStandardDataReduceOptions(
       defaultValue: 'auto',
     });
   }
+
+  if (includeFontSizes) {
+    builder.addNumberInput({
+      path: 'fontSize.title',
+      category: ['Font Size'],
```

Comment:
```suggestion
      category: ['Font size'],
```

Alternative (to be less technical) `Text size` ?